SaneEconomyCore/src/main/java/org/appledash/saneeconomy/utils/JarUtil.java

```diff
@@ -0,0 +1,96 @@
+package org.appledash.saneeconomy.utils;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Files;
+import java.util.List;
+import me.lucko.jarrelocator.JarRelocator;
+import me.lucko.jarrelocator.Relocation;
+
+/**
+ * Some of this code taken from LuckPerms
```

Comment:
Does the license allow this?

Reply:
[LuckPerms is MIT](https://github.com/lucko/LuckPerms/blob/master/LICENSE.txt)

Reply:
Indeed! All of Luck's resources are MIT :D